src/components/LightningConnectWallet.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div>
+    <b-form-select v-model="selectedWallet" class="mb-3">
+      <b-form-select-option
+        :value="{ walletName: 'Select your wallet', type: null, network: null }"
+        disabled
+        >Select your wallet</b-form-select-option
+      >
+      <b-form-select-option
+        :value="{ walletName: 'Zap Desktop', type: 'grpc', network: 'tor' }"
+        >Zap Desktop</b-form-select-option
+      >
+      <b-form-select-option
+        :value="{ walletName: 'Zap Mobile', type: 'rest', network: 'tor' }"
```

Comment:
I think this should be gRPC: https://github.com/getumbrel/umbrel-dashboard/pull/218#issuecomment-698910656

```suggestion
        :value="{ walletName: 'Zap Mobile', type: 'grpc', network: 'tor' }"
```